src/docs/santiment-queries/api-access/index.md

```diff
@@ -37,7 +37,7 @@ On the [API Overview](/sanapi/#overview/) page, you can find information on how
 
 There are two ways to execute queries using the API:
 
-- Directly execute the `computeRawClickhouseQuery` GraphQL query against the API GraphQL endpoint.
+- Directly execute the `computeRawClickhouseQuery` GraphQL request against the API GraphQL endpoint.
```

Comment:
The correct term for `computeRawClickhouseQuery` is a query, it refers to the GraphQL terminology.